contrib/network/connection_balance/dlb/source/connection_balancer_impl.cc

```diff
@@ -15,6 +15,27 @@ namespace Envoy {
 namespace Extensions {
 namespace Dlb {
 
+Envoy::Network::ConnectionBalancerSharedPtr DlbConnectionBalanceFactory::fallback(
+    const envoy::extensions::network::connection_balance::dlb::v3alpha::Dlb::FallbackPolicy
```

Comment:
imo this seems like it should be in anonymous namespace non-member function since it doesn't use any of the member fields

Reply:
Updated.